### PR TITLE
Init cluster admin content with proxies for Indonesian translation

### PR DESCRIPTION
**Pull request by girikuncoro**
This is translation of proxies page into Indonesian language.

/language id

### CODE REVIEW

File: content/id/docs/cluster-administration/proxies.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Berbagai Proxy di Kubernetes
```
Comment (irvifa) [inline content/id/docs/cluster-administration/proxies.md]: nit: <i>Proxy</i>

File: content/id/docs/cluster-administration/proxies.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+{{% capture body %}}
+
+## Berbagai Jenis <i>Proxy</i>
+
+Ada beberapa jenis <i>proxy</i> yang akan kamu temui saat menggunakan Kubernetes:
+
+1.  [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - dijalankan pada <i>desktop</i> pengguna atau di dalam sebuah pod
```
Comment (irvifa) [inline content/id/docs/cluster-administration/proxies.md]: if we're using an API object should we use capital as Pod or pod?

File: content/id/docs/cluster-administration/proxies.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    - melakukan <i>proxy</i> dari alamat localhost ke apiserver Kubernetes
+    - dari <i>client</i> menuju <i>proxy</i> menggunakan HTTP
```
Comment (irvifa) [inline content/id/docs/cluster-administration/proxies.md]: nit: klien

File: content/id/docs/cluster-administration/proxies.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - dari <i>proxy</i> menuju apiserver menggunakan HTTPS
+    - mencari lokasi apiserver
+    - menambahkan <i>header</i> autentikasi
+
+1.  [apiserver proxy](/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services):
+
+    - merupakan sebuah <i>bastion</i> yang ada di dalam apiserver
+    - menghubungkan pengguna di luar kluster ke alamat-alamat IP di dalam kluster yang tidak bisa terjangkau
+    - dijalankan bersama <i>process-process</i> apiserver
+    - dari <i>client</i> menuju <i>proxy</i> menggunakan HTTPS (atau http jika dikonfigurasi pada apiserver)
```
Comment (irvifa) [inline content/id/docs/cluster-administration/proxies.md]: nit: klien

File: content/id/docs/cluster-administration/proxies.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+1.  Sebuah <i>Proxy/Load-balancer</i> di depan satu atau banyak apiserver:
+
+    - keberadaan dan implementasinya bervariasi tergantung pada kluster (contohnya nginx)
+    - ada di antara seluruh <i>client</i> dan satu/banyak apiserver
+    - jika ada beberapa apiserver, berfungsi sebagai <i>load balancer</i>
+
+1.  <i>Cloud Load Balancer</i> pada servis eksternal:
+
+    - disediakan oleh beberapa penyedia layanan cloud, seperti AWS ELB, Google Cloud Load Balancer
+    - dibuat secara otomatis ketika Service dari Kubernetes punya <i>type</i> `LoadBalancer`
```
Comment (irvifa) [inline content/id/docs/cluster-administration/proxies.md]: dengan tipe

File: content/id/docs/cluster-administration/proxies.md (hunk 2 above)
Comment (girikuncoro) [reply content/id/docs/cluster-administration/proxies.md]: true, Pod is better

File: content/id/docs/cluster-administration/proxies.md (hunk 1 above)
Comment (girikuncoro) [reply content/id/docs/cluster-administration/proxies.md]: not possible to do italic on title template

**Pull request merged**